### DISCUSSION TITLE
Await for more than one signal ?

### DISCUSSION THREAD

**Discussion by tebz** [godotengine/godot-proposals#6551] [Scripting]
It already exists one
https://github.com/godotengine/godot-proposals/issues/6243

Is there a easy way to AWAIT for more than one signal ?
Something like:
_	var b = await %Button1.button_up | %Button2.button_up
_	print("button_up " + str(b) ) 
in stead of:
_	var b = await b1_or_b2
_	print("button_up " + str(b) ) 
_
func _on_button_1_button_up():
_	emit_signal("b1_or_b2", button.b1)
_
func _on_button_2_button_up():
_	emit_signal("b1_or_b2", button.b2)

**Comment by AThousandShips** [comment]
Already tracked by #6243

**Comment by dalexeev** [comment]
Let's close for consolidation, since there is already a proposal. Thanks for the contribution nonetheless!
